controllers/suite_test.go

```diff
@@ -40,16 +44,21 @@ import (
 var cfg *rest.Config
 var k8sClient client.Client
 var testEnv *envtest.Environment
+var clusterReconciler *HStreamDBReconciler
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecs(t, "Controller Suite")
+	RunSpecs(t, "HStreamDB Controller")
 }
 
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
+	if isUsingExistingCluster() {
+		Expect(os.Setenv("USE_EXISTING_CLUSTER", "true")).To(Succeed())
+	}
```

Comment:
If `isUsingExistingCluster` return true, means already set `USE_EXISTING_CLUSTER`, so why set it again in this?